### PR TITLE
Install: Schedule multipath before scc-registration

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/87710
http://kazhua.qa.suse.de/tests/182
http://kazhua.qa.suse.de/tests/183

### CODE REVIEW

**Review by rwx788 — CHANGES_REQUESTED** [review]
This change will break online medium test. Michael Grifalconi has contacted us yesterday and I have confirmed that this differs for those 2 mediums. In case you want to run this test suite for the full medium, you should either use conditional schedule or separate schedule.

**Comment by asdil12** [comment]
I updated the schedule to schedule the `multipath` module before or after `scc_registration` according to `FLAVOR`.

File: schedule/yast/multipath.yaml [hunk 1]
```diff
@@ -3,15 +3,25 @@ description:    >
...
   Tests successful detection of multipath and installation and
   validate multipath configuration and tools after booting.
+conditional_schedule:
+  multipath_on_full_medium:
+    FLAVOR:
+      'Full-QR':
+        - installation/multipath
+  multipath_on_online_medium:
+    FLAVOR:
+      'Online-QR':
```
Comment (rwx788) [inline schedule/yast/multipath.yaml:13]: We run this test for just `Online` flavor, so please add condition for it too. See https://openqa.suse.de/tests/5317550

Comment (rwx788) [reply schedule/yast/multipath.yaml:13]: Also, please put a comment explaining this condition at least.

Comment (asdil12) [reply schedule/yast/multipath.yaml:15]: I added `Full` and `Online`

**Comment by asdil12** [comment]
Added a comment and verification runs.

**Review by rwx788 — APPROVED** [review]

**Pull request merged**